pgo/cmd/create.go

```diff
@@ -322,6 +337,24 @@ func init() {
 	CreateCmd.AddCommand(createNamespaceCmd)
 
 	// flags for "pgo create cluster"
+	createClusterCmd.Flags().StringSliceVar(&Annotations, "annotation", []string{},
+		"Add an Annotation to all of the managed deployments (PostgreSQL, pgBackRest, pgBouncer)"+
+			"The format to add an annotation is \"name=value\"\n"+
+			"The format to remove an annotation is \"name-\"\n\n"+
+			"For example, to add two annotations:\n"+
+			"--annotation=hippo=awesome --annotation=elephant=cool")
+	createClusterCmd.Flags().StringSliceVar(&AnnotationsBackrest, "annotation-pgbackrest", []string{},
+		"Add an Annotation to specifically to pgBackRest deployments"+
+			"The format to add an annotation is \"name=value\"\n"+
```

Comment:
Looks there's an extra `to` in this description before `specifically` (and same in those update commands that follow),  e.g. should be:

```
Add an Annotation specifically to pgBackRest deployments
```